### ISSUE TITLE
[Bug]: lzop not working on arm

### ISSUE THREAD

**Issue by sk0kanik** [termux/termux-packages#7602]
### Problem description

lzop shows "Illegal instruction" and do not compress file properly

### What steps will reproduce the bug?

install lzop using pkg

then try to compress
```
~ $ du -ha
48K     ./file
52K     .
~ $ lzop file
Illegal instruction
~ $ echo $?
132
~ $ du -ha
48K     ./file
4.0K    ./file.lzo
56K     .
~ $ lzop -t file.lzo
testing file.lzo
lzop: unexpected end of file: file.lzo
```

### What is the expected behavior?

successfully compress the file

### System information

termux-info:

```
Application version:
0.117
Packages CPU architecture:
arm
Subscribed repositories:
# sources.list
deb https://packages.termux.org/apt/termux-main/ stable main
# game-repo (sources.list.d/game.list)
deb https://packages.termux.org/apt/termux-games games stable
# science-repo (sources.list.d/science.list)
deb https://packages.termux.org/apt/termux-science science stable
Updatable packages:
All packages up to date
Android version:
9
Kernel build information:
Linux localhost 4.4.147 #1 SMP PREEMPT Fri Mar 13 14:51:18 CST 2020 armv7l Android
Device manufacturer:
Symphony
Device model:
Symphony G10
```

**Comment by xtkoba** [comment]
I have found an issue of unaligned memory accesses for arm, which should be resolved by #7603. I'm not sure if that is relevant to the main issue here, as an unaligned access typically causes a bus error and not an illegal instruction. But please try the patch first.

**Comment by sk0kanik** [comment]
> please try the patch first.
I have tried. After this fix, lzop is working. Thank you.
